pygmt/src/plot.py

```diff
@@ -118,9 +118,9 @@ def plot(self, data=None, x=None, y=None, size=None, direction=None, **kwargs):
         *dx/dy* [Default is no offset]. If *dy* is not given it is set
         equal to *dx*.
     error_bar : bool or str
-        [**+b**\|\ **d**\|\ **D**][**+xl**\|\ **r**\|\ *x0*]\
-        [**+yl**\|\ **r**\|\ *y0*][**+p**\ *pen*].
-        Draw symmetrical error bars. Full documentation is at
+        [**x**\|\ **y**\|\ **X**\|\ **Y**][**+a**\|\ **A**]\
+        [**+cl**\|\ **f**][**+n**][**+w**\ *cap*][**+p**\ *pen*].
+        Draw error bars. Full documentation is at
         :gmt-docs:`plot.html#e`.
```

Comment:
@seisman Should I update the syntax for `error_bar` corresponding to the change in https://github.com/GenericMappingTools/gmt/pull/7072? Or should we keep the syntax related to GMT 6.4.0 (for the case PyGMT v0.8.0 will be released before GMT 6.5.0)? 

```suggestion
        [**x**\|\ **y**\|\ **X**\|\ **Y**][**+a**\|\ **A**]\
        [**+cl**\|\ **f**][**+n**][**+w**\ *width*\ [/*cap*]]\
        [**+p**\ *pen*].
        Draw error bars. Full documentation is at
        :gmt-docs:`plot.html#e`.
```

Reply:
I think we should keep the old syntax because GMT 6.5.0 is not released yet.

Reply:
That's fine with me!